fdbclient/Schemas.cpp

```diff
@@ -527,6 +527,10 @@ const KeyRef JSONSchemas::statusSchema = R"statusSchema(
          "lock_uid": "00000000000000000000000000000000"
       },
       "generation":2,
+      "gray_failure" : {
+         "excluded_servers" : [
+         ]
```

Comment:
Maybe add a sample here, since this is a pair for each server:
```
        "excluded_servers" : [
             {
                  "address": "127.0.0.1:4500",
                  "time": 1731294251
              }
         ]
```

Reply:
Done